python/.changelog.d/2191.fixed

```diff
@@ -0,0 +1 @@
+Fixed broken links in docs.
```

Comment:
Wouldn't add this to changelog - you can silence the failing CI job by adding `[no changelog]` to commit message, or you can ignore the failing job.